celiagg/vertex_source.h

```diff
@@ -89,9 +89,9 @@ class PathSource : public VertexSource
     unsigned last_vertex(double* x, double* y) const;
 
     template<class VertexSource>
-    void concat_path(VertexSource& vs, unsigned path_id)
+    void concat_path(VertexSource& vs)
     {
-        m_path.concat_path(vs, path_id);
+        m_path.concat_path(vs);
```

Comment:
The way that `agg::path_storage::concat_path` is used by this code doesn't require the second argument, which sets the starting vertex index in the source which is being added to the receiver. It's always 0 because we want to copy the whole source from its start.